.azure-pipelines/azure-pipelines-win.yml

```
@@ -14,31 +14,16 @@ jobs:
   timeoutInMinutes: 360
   variables:
     CONDA_BLD_PATH: D:\\bld\\
+    MINIFORGE_HOME: D:\Miniforge
```

Try changing this to see if it helps
```suggestion
    MINIFORGE_HOME: C:\Miniforge
```